schema_based_on_diagram.sql

```diff
@@ -0,0 +1,77 @@
+-------create patients table from the ERD-------
+CREATE TABLE IF NOT EXISTS public.patients
+(
+    id integer NOT NULL GENERATED BY DEFAULT AS IDENTITY ( INCREMENT 1 START 1 MINVALUE 1 MAXVALUE 2147483647 CACHE 1 ),
+    name character varying COLLATE pg_catalog."default",
+    date_of_birth date,
+    CONSTRAINT patients_pkey PRIMARY KEY (id)
+)
+
+--------create treatments table ---------
+CREATE TABLE IF NOT EXISTS public.treatments
```

Comment:
- [ ] Please create indexes for all FOREIGN KEYS.

```ruby 
* CREATE INDEX ON medical_histories (patient_id);
* CREATE INDEX ON invoices (medical_history_id);
* CREATE INDEX ON invoice_items (invoice_id);
* CREATE INDEX ON invoice_items (treatment_id);
* CREATE INDEX ON medical_histories_has_treatments (medical_history_id);
* CREATE INDEX ON medical_histories_has_treatments (treatment_id);

Reply:
Well resolved 👍🏽